adagrams/game.py

```diff
@@ -1,11 +1,121 @@
+#from tests.test_wave_01 import LETTER_POOL
+import random
+import copy
+LETTER_POOL = {
```

Comment:
👍 I do like having these giant data structures not cluttering up a function. Naming the variable like a constant helps us communicate that we should be careful not to do anything that would alter the data.